manuscript/chapters/handbook-01-introduction.qmd

```diff
@@ -0,0 +1,51 @@
+# Einleitung
```

Comment:
This is a roadmap, right? If so, we need to merge to something else than `main` (`dev`for example). `main` is for production ready code only.